qiita_db/portal.py

```diff
@@ -40,10 +40,12 @@ def __init__(self, portal):
     @staticmethod
     def list_portals():
         """Returns list of portals available in system
+
         Returns
         -------
         list of str
             List of portal names for the system
+
         Notes
         -----
         This does not return the QIITA portal in the list, as it is a required
```

Comment:
I did this because the QIITA portal is a default portal that you can't edit. Would it make it clearrer to change the docstring to `...list of non-default portals...`?